classes/options-general.php

```diff
@@ -564,13 +564,13 @@ class='howto'><?php _e( 'Select "No Limit" to display all results within the sea
 
 													<tr valign="top">
 														<td><label
-																for="map_pages"><?php _e( 'Map Page IDs', 'simplemap' ); ?></label>
+																for="map_pages"><?php _e( 'Map Page IDs', 'SimpleMap' ); ?></label>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 131 characters

<!-- Comment by [SideCI](https://sideci.com) -->